From:	Kamal Mostafa <kamal@canonical.com>
To:	Linux Kernel <linux-kernel@vger.kernel.org>,
	intel-gfx@lists.freedesktop.org
Cc:	Matthew Garrett <mjg@redhat.com>,
	Keith Packard <keithp@keithp.com>,
	Kamal Mostafa <kamal@canonical.com>
Subject: [PATCH] i915: do not setup intel_backlight twice
Date:	Mon, 22 Aug 2011 12:39:10 -0700
Message-Id: <1314041950-12433-1-git-send-email-kamal@canonical.com>
X-Mailer: git-send-email 1.7.4.1
In-Reply-To: <20110817141010.GA21479@redhat.com>
References: <20110817141010.GA21479@redhat.com>
Sender:	linux-kernel-owner@vger.kernel.org
Precedence: bulk
List-ID: <linux-kernel.vger.kernel.org>
X-Mailing-List:	linux-kernel@vger.kernel.org

The commit "Not all systems expose a firmware or platform mechanism for
changing the backlight intensity on i915, so add native driver support"
adds calls to  intel_panel_setup_backlight() from intel_{lvds,dp}_init
so do not call it again from intel_setup_outputs().

BugLink: http://bugs.launchpad.net/bugs/831542

Signed-off-by: Kamal Mostafa <kamal@canonical.com>
---
 drivers/gpu/drm/i915/intel_display.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index ee1d701..5a1ae9f 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -7238,8 +7238,6 @@ static void intel_setup_outputs(struct drm_device *dev)
 			intel_encoder_clones(dev, encoder->clone_mask);
 	}
 
-	intel_panel_setup_backlight(dev);
-
 	/* disable all the possible outputs/crtcs before entering KMS mode */
 	drm_helper_disable_unused_functions(dev);
 }
-- 
1.7.4.1
